README.md

```diff
@@ -29,3 +29,7 @@ What is the asymptotic runtime ($\Theta$) of your algorithm in terms of the
 number of comparisons? What is it in terms of the number of flips? Add your
 answer to this markdown file.
 
+I worked on some of this assignment with Megan, Olivia, and Ashlyn.
+
+The asymptotic runtime for the number of comparisons should be big Theta n^2. It should be this way based on the way i wrote it because it takes n - the sorted part of the array, and each of the sorted part of the array gets smaller as it gets more sorted summing up to another n making it n^2.
+The asymptotic runtime for the number of flips should be big Theta n^2 aswell. It should be this way because the way i wrote it, the unsorted array flips twice, once it flips the largest element in the unsorted part to the front, and then it flips it into the correctly sorted position. The worst case it would look at each element in the unsorted part of the array giving in time n and it has to find the right position to place the largest element going through the array again making it big theta of n^2.
```

Comment:
Are you doing a flip for every comparison?

Reply:
no but it will flip twice everytime it goes through the unsorted array which will be the whole array for worst case

Reply:
I dont think so, they still have the same time complexity though the way that im thinking about it, because a flip has to iterate through the unsorted array each time giving linear time and then that will be iterated linear amount of time because of the while loop which goes until its sorted.

Reply:
Have another look at your code -- are they really the same asymptotically?

Reply:
oh is it actually n^2 because there are two for loops iterating through the entire array?

Reply:
I think you're just guessing at this point. This is the original answer you gave, which is not correct.

Reply:
Ive been going through this multiple times and line by line and im still getting O(n^2).  While it is more complicated, it simplifies to be O(n^2) because the other values are arbitrary.  Flipping takes O(n) and the actual sorting takes O(n^2) so the n^2 is dominant giving us that as the overall complexity.

Reply:
But how many flips and how many comparisons are you making?

Reply:
worse case i make two flips maximum for each element so i would be making 2n flips and for comparisons id be making n(n-1)/2 comparisons because the algorithm is looking for the largest element and for every iteration of the while loop the unsorted part is n-k which requires (n - k - 1) comparisons.  Summing that simplifies to n(n-1)/2

Reply:
i updated my answer